symbols.xml

```diff
@@ -2226,7 +2225,7 @@
         <global-address name='enabler'/>
         <global-address name='flows'/>
         <global-address name='gps'/>
-        <global-address name='gview'/>
+        <global-address name='gview' value='0x1421040'/>
```

Comment:
They each had two entries in their respective symbol tables, so I kept the one that matched the example symbol-table in the comment at the top of the file.